python/tank/platform/software_launcher.py

```diff
@@ -234,6 +234,21 @@ def shotgun(self):
         """
         return self.sgtk.shotgun
 
+    def _get_minimum_supported_version(self):
+        """
+        The minimum software version that is supported by the launcher.
+        Returned as a string, for example "2015" or "2015.3.sp3".
+        Returns ``None`` if no constraint has been set.
+        Also see related method :meth:`~is_version_supported`.
+        """
+        return self.__minimum_supported_version
+
+    def _set_minimum_supported_version(self, version):
```

Comment:
Could you elaborate why a setter is required here? From my understanding this interface is implemented by each engine and as such the implementer should provide a _get_minimum_support_version method that the getter would tap into. When is setting the minimum version from outside the launcher a requirement?